e2e/tests/app.e2e.js

```diff
@@ -0,0 +1,44 @@
+import { expect, test } from '@playwright/test';
+import { NavBarPage } from '../pages/nav-bar-page';
+import { HomePage } from '../pages/home-page';
+
+test.describe('AngularJS-Material-Vite app', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+  test.describe('TopNav', () => {
+    test('Navbar', async ({ page }) => {
+      const navBarPage = new NavBarPage(page);
+
+      await expect(navBarPage.logo).toHaveAttribute('src', '/assets/img/icon-ng-hotrod-light.svg');
+      await expect(navBarPage.header).toHaveText('ng-hotrod');
+    });
+    test('Redirect to GitHub page', async ({ page }) => {
+      const navBarPage = new NavBarPage(page);
+
+      await navBarPage.clickOnGitHubIcon();
+
+      await expect(navBarPage.githubRepoName).toHaveText('angularjs-material-vite');
```

Comment:
FWIW, I don't think this is a very reliable way to assert that the GitHub icon works.
It is possible that the link goes to a different page that also happens to have an `agularjs-material-vite` link (or n the future, it could be that our page has such a link too).

I believe using the URL is much more straight-forward and reliable way to assert that.

Reply:
I changed the assertions to expect a specific page URL after the redirect

<img width="608" alt="image" src="https://user-images.githubusercontent.com/5073455/223716421-44744739-15a6-44e7-be40-8b97ab88dd3b.png">